tests/run_tests

```diff
@@ -26,6 +26,7 @@ function testMPI()
 }
 
 testSerial ./SimpleDataTest.cpp.out "Testing simple data read/write..."
+testSerial ./readBool.py "Testing h5py compatible read..."
```

Comment:
Would it make sense to check if python is available and run the readBool.py only in that case?